cumulusci/tasks/robotframework/robotframework.py

```diff
@@ -108,6 +108,9 @@ def _run_task(self):
         options["outputdir"] = os.path.relpath(
             os.path.join(self.working_path, options.get("outputdir", ".")), os.getcwd()
         )
+        # Set as a return value so other things that want to use
+        # this file (e.g. MetaCI) know where it is
+        self.return_values["robot_outputdir"] = options["outputdir"]
```

Comment:
This looks like a great enhancement. I didn't know `self.return_values` was a thing.  I guess I should find time over the holidays to RTFM.

Reply:
@boakley I think you'll probably find the FM sadly lacking... Return values are not displayed anywhere, but are available for later flow steps to reference in their options (i.e. there could be a "process_robot_output" step which uses this return value:
```
steps:
    1:
        task: robot
    2:
        task: process_robot_output
        options:
            path: ^^robot.robot_outputdir
```

But the thing that's prompting the addition of this particular return value is wanting to use it in the system that is running the flow (MetaCI), rather than by another task.